app/scripts/background.js

```diff
@@ -76,7 +75,7 @@ import DesktopManager from '@metamask/desktop/dist/desktop-manager';
 
 // Setup global hook for improved Sentry state snapshots during initialization
 const inTest = process.env.IN_TEST;
-const localStore = inTest ? new ReadOnlyNetworkStore() : new LocalStore();
```

Comment:
Even in test mode, we want to use an empty `new LocalStore()`, otherwise if we use `new ReadOnlyNetworkStore()` here, it will initiate local storage, then it cannot detect that user installs app the first time and the app installed metric will not be sent.
As this _localStore_ value only be used once in this file, so it's safe to change.